.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+[Note: Please fill out the following template for your pull request. lines
+tagged with "Note" can be removed from the template.]
+
+[Note: Before this PR can be reviewed please agree to the CLA covering this
+repo. Please also review the contribution guide -
+https://github.com/openconfig/public/blob/master/doc/external-contributions-guide.md]
+
+### Change Scope
+
+* [Please briefly describe the change that is being made to the models.]
+* [Please indicate whether this change is backwards compatible.]
+
```

Comment:
* [Note: Consider adding a snippet of pyang output showing the model changes.  This can be in tree or flattened format.  Be sure to include yang augments that affect the model output.]

```suggestion

```